src/actions/bridgeETH.ts

```diff
@@ -0,0 +1,36 @@
+import {
+  Address,
+  Chain,
+  Transport,
+  Account,
+  WalletClient,
+  SendTransactionParameters,
+} from 'viem'
+import { OpChainL2 } from '@roninjin10/rollup-chains'
+import { bridgeSendTransaction } from './bridgeSendTransaction'
+
+// Currently hardcoded to existing sdk value
+// consider optimizing in future
+const ETH_TRANSFER_L2_GAS = BigInt(200_000)
+
+export async function bridgeETH<
+  TChainL1 extends Chain | undefined = Chain,
+  TChainL2 extends OpChainL2 | undefined = OpChainL2,
+  TAccount extends Account | undefined = Account | undefined,
+  TChainOverride extends Chain | undefined = Chain | undefined,
+>(
+  client: WalletClient<Transport, TChainL1>,
+  { data, value, toChain, account, toAccount, ...restArgs }: { toChain: TChainL2, toAccount: Address | Account, value: bigint } & SendTransactionParameters<TChainL1, TAccount, TChainOverride>,
+): Promise<string> {
+  const to = toAccount ?? account ?? client.account?.address
+  // TODO better typescriptin
+  return bridgeSendTransaction(client as any, {
```

Comment:
why use `bridgeSendTransactions` rather than calling the standard bridge?

Reply:
Yea should change to standard bridge just so it serves as a better reference. This shouldn't block any other prs or cause merge conflicts so it can happen in this pr or it can happen in a future pr no big difference either way